utils/build-script-impl

```diff
@@ -2511,11 +2509,9 @@ for host in "${ALL_HOSTS[@]}"; do
                       -DLLDB_SWIFTC:PATH="$(build_directory ${LOCAL_HOST} swift)/bin/swiftc"
                       -DLLDB_SWIFT_LIBS:PATH="$(build_directory ${LOCAL_HOST} swift)/lib/swift"
                       -DCMAKE_INSTALL_PREFIX:PATH="$(get_host_install_prefix ${host})"
-                      -DLLDB_PATH_TO_LLVM_SOURCE:PATH="${LLVM_SOURCE_DIR}"
-                      -DLLDB_PATH_TO_CLANG_SOURCE:PATH="${CLANG_SOURCE_DIR}"
+                      -DClang_DIR:PATH=${llvm_build_dir}/lib/cmake/clang
```

Comment:
Any reason this isn't all caps?

Reply:
Yes!  That is the CMake module name `Clang`.  Using `CLANG_DIR` will result in CMake not finding the `Clang` package.

Reply:
Ok, just wanted to clarify if this was a CMake constraint or style choice.